redbot/cogs/filter/filter.py

```diff
@@ -100,44 +100,50 @@ async def _filter(self, ctx: commands.Context):
         """Add or remove words from server filter.
 
         Use double quotes to add or remove sentences.
-
-        Using this command with no subcommands will send the list of
-        the server's filtered words.
         """
-        if ctx.invoked_subcommand is None:
-            server = ctx.guild
-            author = ctx.author
-            word_list = await self.config.guild(server).filter()
-            if word_list:
-                words = ", ".join(word_list)
-                words = _("Filtered in this server:") + "\n\n" + words
-                try:
-                    for page in pagify(words, delims=[" ", "\n"], shorten_by=8):
-                        await author.send(page)
-                except discord.Forbidden:
-                    await ctx.send(_("I can't send direct messages to you."))
+        pass
+
+    @_filter.command(name="list")
+    async def _global_list(self, ctx: commands.Context):
+        """Send a list of this servers filtered words."""
+        server = ctx.guild
+        author = ctx.author
+        word_list = await self.config.guild(server).filter()
+        if not word_list:
+            await ctx.send(_("There is no current words setup to be filtered in this server."))
+            return
+        words = ", ".join(word_list)
```

Comment:
Hey @flaree  can you please use https://docs.discord.red/en/latest/framework_utils.html#redbot.core.utils.chat_formatting.humanize_list here?

```suggestion
        words = humanize_list(word_list)
```

You will also need to edit the import to import `humanize_list` from `redbot.core.utils.chat_formatting`